scripts/copy_bundles.py

```diff
@@ -50,7 +53,7 @@ class CopyBundle(DeferredTaskExecutor):
 
     def main(self):
         if self.args.shared:
-            with azul.dss.shared_credentials():
+            with AWS().service_account_credentials():
```

Comment:
```suggestion
            with aws.service_account_credentials():
```
Repeat in all places `AWS().service_account_credentials()` is used.